x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/settings/hooks/api.ts

```diff
@@ -14,6 +14,20 @@ export const getIlmPolicies = async (): Promise<PolicyFromES[]> => {
   return await apiService.get('/api/index_lifecycle_management/policies');
 };
 
+export const getDslPolicies = async (): Promise<any> => {
```

Comment:
Should we type it properly?

Reply:
7008b1b929dbf523a433f00fc5a054f838c1bd5e